deploy/helm/docs/install-mongodb-operator.md

```
@@ -0,0 +1,277 @@
+### Verify the install
+
+```bash
+kubectl get pods -n appsmith
+kubectl get mongodbcommunity -n appsmith
+```
+
+Expected output (abridged):
+
+```
+NAME                              READY   STATUS
+appsmith-0                        1/1     Running
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)